src/pages/partners/_partnership-opportunities.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { Header, Text, CardStyle } from 'components/elements'
+import { localize, LocalizedLink } from 'components/localization'
+import { Container, SectionContainer, Flex } from 'components/containers'
+// import device from 'themes/device'
+// Icons
+import PartnerAffiliate from 'images/svg/partner-affiliate.svg'
+import PartnerPaymentAgent from 'images/svg/partner-payment-agent.svg'
+
+const ClientCard = styled(LocalizedLink)`
+    ${CardStyle}
+    text-decoration: none;
+    max-width: 38.4rem;
+    padding: 3.2rem;
+    min-height: 23.7rem;
+    position: relative;
+    overflow: hidden;
+    transition: transform 0.3s;
+    margin: 0 1.2rem;
+    cursor: pointer;
+
+    &:first-child {
+        margin-left: 0;
+    }
+    &:last-child {
+        margin-right: 0;
+    }
+    & svg {
+        width: 24px;
+        height: 24px;
+    }
+    &:hover {
+        transform: translateY(-1.5rem);
+        ${(props) =>
+            props.first
+                ? css`
+                      svg > g > path:first-child {
+                          fill: var(--color-red);
+                      }
+                  `
+                : css`
+                      svg > g > g > path {
+                          fill: var(--color-red);
+                      }
+                      svg > g > g > g > path:last-child {
```

Comment:
:smile: :+1:

Reply:
the svg structure, is so fucked up here. both of the svg has different structure, i dont know any other way to do it 🙏